pkg/render/fluentd.go

```diff
@@ -455,6 +455,10 @@ func (c *fluentdComponent) container() corev1.Container {
 		StartupProbe:    c.startup(),
 		LivenessProbe:   c.liveness(),
 		ReadinessProbe:  c.readiness(),
+		Ports: []corev1.ContainerPort{{
```

Comment:
Is there or should there be a service that is targeting this?

Reply:
No I'm going to use a pod selector from prometheus.